prow/cmd/crier/main.go

```diff
@@ -159,7 +157,7 @@ func main() {
 			controllers,
 			crier.NewController(
 				prowjobClient,
-				queue,
+				kube.RateLimiter(controllerName),
```

Comment:
Hm - the controllerName should be unique between all the queues. Assuming this gets plumbed down as the name of the workqueue...

Reply:
👍 good catch

Reply:
done